client/src/components/Maps.js

```diff
@@ -67,12 +67,14 @@ function Maps() {
                         loadingElement={<div style={{ height: '100%' }} />}
                         containerElement={<div style={{ height: '100%' }} />}
                         mapElement={<div style={{ height: '100%' }} />}
+                        businessesObj={businesses}
```

Comment:
where is `businessesObj` prop being accessed?

Reply:
It is not being used anymore, it is okay to delete and I will do so.